<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
qproperty-alignment:'AlignVCentr | AlignRight';
border: 1xp solid gray;
}
backgroynd-color : white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="displayLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
qproperty-alignment: 'AlignVCenter | AlignRight';
border-bottom: 1xp solidgray;
}
background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInterest">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(215,215,215);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
} </string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(215,215,215);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
} </string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEqual">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>288</y>
      <width>53</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255,151,57);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>115</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255,151,57);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_negNumb">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(215,215,215);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
} </string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255,151,57);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_divide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255,151,57);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(255,151,57);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(115,115,115);
border: 1xp solid gray;
}
QPushButton:pressed
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2:1, stop: 0 #dadbde, stop: 1 #f6f7fa);
} 

</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
